src/main/java/com/asem/regestration/model/dao/impl/StudentDAOIMpl.java

```diff
@@ -32,14 +32,6 @@ public Connection getConnection() throws SQLException {
         return dataSourcePool.getConnections().getConnection();
 
     }
-
-
-
-
-
-
-
-
     @Override
     public Student findById(String id) throws SQLException {
         String sql="select *from student2.student where student.id="+id+"";
```

Comment:
Use String.format instead of concatenating string
What does the last "" after id+ do ?